lib/bolt/outputter/human.rb

```diff
@@ -112,6 +116,27 @@ def print_start(target)
         @stream.puts(colorize(:green, "Started on #{target.safe_name}..."))
       end
 
+      def print_container_result(result)
+        if result.success?
+          @stream.puts(colorize(:green, "Finished running container #{result.object}:"))
+        else
+          @stream.puts(colorize(:red, "Failed running container #{result.object}:"))
+        end
+
+        if result.error_hash
+          @stream.puts(colorize(:red, remove_trail(indent(2, result.error_hash['msg']))))
+        end
+
+        # Only print results if there's something other than empty string and hash
+        if result.stdout.empty? && result.stderr.empty?
+          @stream.puts(indent(2, "Running container #{result.object} completed successfully with no result"))
+        elsif result.stdout.empty?
+          @stream.puts(indent(2, "STDERR: #{result.stderr}"))
+        else
+          @stream.puts(indent(2, result.stdout))
+        end
```

Comment:
I think this should print both stdout and stderr, if available. Since we don't interleave the output, something similar to what we used to do for commands/scripts would work: https://github.com/puppetlabs/bolt/blob/3.3.0/lib/bolt/outputter/human.rb#L147-L154

Reply:
Oh ha - I hadn't rebased so didn't see that code in the file